Docs review (#344)
archway-network/archway-docs: content/3.community/4.governance/5.vote.md

## How to vote on governance proposals

> The easiest way to vote on governance proposals is through [Archway Connect](/community/archway-connect). Anyways, you can vote on governance proposals also via a [wallet](wallet.md) or the [archwayd](../developers/developer-tools/daemon) directly.

Reviewer: ```suggestion
The easiest way to vote on governance proposals is through [Archway Connect](/community/archway-connect). However, you can also vote on governance proposals through a [wallet](wallet.md) or directly via [archwayd](../developers/developer-tools/daemon).
```

### Voting on governance proposals via Archway Connect

- Connect to Archway connect by pressing **Connect your wallet**:
![](/images/docs/ac_vote_1.png)
and select the wallet you want to connect with:
![](/images/docs/ac_vote_2.png)

Then select the **Vote** section, on the side menu:
![](/images/docs/ac_vote_3.png)

> Look at the governance proposals that are in the **Voting** phase, and click on it.

Reviewer: ```suggestion
Review the governance proposals currently in the **Voting** phase and click on the desired proposal for more details.
```

![](/images/docs/ac_vote_4.png)

> Read at the governance proposal's details,and click on **Vote**:

Reviewer: ```suggestion
Read the governance proposal's details, and click on **Vote**:
```
